cmd/onewire-list/main.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2017 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+// Onewire-list lists all onewire buses and devices.
+
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"periph.io/x/periph/conn/onewire/onewirereg"
+	"periph.io/x/periph/host"
+)
+
+func mainImpl() error {
+	if _, err := host.Init(); err != nil {
+		return err
+	}
+	for _, ref := range onewirereg.All() {
+		fmt.Println("BUS name:", ref.Name)
```

Comment:
Can you make its output closer to https://github.com/google/periph/blob/master/cmd/spi-list/main.go and https://github.com/google/periph/blob/master/cmd/i2c-list/main.go ?

In particular the bus number, and print the Q pin. Obviously the Search() is onewire specific. That said I'm also open to tweak i2c-list and spi-list if you have improvement ideas.

Reply:
I tried to align with the other commands. I have no improvement idea in general.